cdapython/results/result.py

```diff
@@ -154,15 +136,17 @@ def has_next_page(self) -> bool:
             bool: returns a bool value if there is a next page
         """
         if isinstance(self._offset, int) and isinstance(self._limit, int):
-            return (self._offset + self._limit) <= self.total_row_count
+            return (self._offset + self._limit) < self.total_row_count
```

Comment:
If these values are no longer optional, do we really need these checks in the statement above?